grpc-stubs/aio.pyi

```diff
@@ -0,0 +1,258 @@
+import typing
+from concurrent import futures
+from types import TracebackType
+from . import (
+    _Options,
+    Compression,
+    GenericRpcHandler,
+    ServerCredentials,
+    StatusCode,
+    ChannelCredentials,
+    CallCredentials,
+)
+
+"""Create Client"""
+
+# FIXME
+ClientInterceptor = typing.Any
+
+def insecure_channel(
+    target: str,
+    options: typing.Optional[_Options] = None,
+    compression: typing.Optional[Compression] = None,
+    interceptors: typing.Optional[typing.Sequence[ClientInterceptor]] = None,
+) -> Channel: ...
+def secure_channel(
+    target: str,
+    credentials: ChannelCredentials,
+    options: typing.Optional[_Options] = None,
+    compression: typing.Optional[Compression] = None,
+    interceptors: typing.Optional[typing.Sequence[ClientInterceptor]] = None,
+) -> Channel: ...
+
+"""Create Server"""
+
+def server(
+    migration_thread_pool: typing.Optional[futures.Executor] = None,
+    handlers: typing.Optional[typing.Sequence[GenericRpcHandler]] = None,
+    interceptors: typing.Optional[typing.Sequence[typing.Any]] = None,
+    options: typing.Optional[_Options] = None,
+    maximum_concurrent_rpcs: typing.Optional[int] = None,
+    compression: typing.Optional[Compression] = None,
+) -> Server: ...
+
+"""Channel Object"""
+
+RequestSerializer = typing.Callable[[typing.Any], bytes]
```

Comment:
The versions of these types in the non-async version  of this package are less specific than this. It looks like I have a note to investigate where they actually come from a bit further. What evidence did you use to get the types for these? Does it apply to the non-async version too?

Reply:
The aio docs at https://grpc.github.io/grpc/python/grpc_asyncio.html#grpc.aio.Channel.stream_stream suggest this is the appropriate type. The non-async equivalents lack that typing hint in the docs, but it's probably a safe assumption that the types are the same. That said, I don't want to create incorrect definitions or blow up this PR, so I'd rather that were taken care of in a separate PR.

Reply:
Happy to leave changes to the non-async version. Please add that link with a comment prefixed `# XXX:` explaining why async is typed and the sync version isn't and include the link.